packages/pluggableWidgets/line-chart-native/src/utils/__tests__/StyleMappers.spec.ts

```diff
@@ -0,0 +1,124 @@
+import { mapToAxisStyle, mapToGridStyle, mapToLineStyle, mapToMarkerStyle } from "../StyleMappers";
+import { LineChartAxisStyle, LineChartLineStyle } from "../../ui/Styles";
+
+describe("StyleMappers", () => {
+    describe("mapToGridStyle", () => {
+        it("maps the relevant styles", () => {
+            const gridStyle = {
+                backgroundColor: "orange",
+                dashArray: "5,5",
+                lineColor: "transparent",
+                paddingBottom: 30,
```

Comment:
Same question about paddingVertical, horizontal and padding.. If that is the case we also need to test them here.

Reply:
Added them here, to verify that mappers don't include these styles.